uma/uma.py

```diff
@@ -792,3 +798,58 @@ def _parse_timestamp(
         return datetime.fromtimestamp(timestamp, timezone.utc)
     except ValueError as ex:
         raise InvalidRequestException("Invalid timestamp in request.") from ex
+
+
+def create_uma_invoice(
+    receiver_uma: str,
+    amount: int,
```

Comment:
Maybe use `receiving_currency_amount` here just to be explicit (to avoid confusing with sending amount or sats amount).